### PR TITLE
Add Nice G.O. integration

### PR DESCRIPTION
**Pull request by IceBotYT**
## Proposed change

Adds documentation for Nice G.O. integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/122748
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/5709
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced the "Nice G.O." integration, enabling control of Nice G.O. garage doors and lighting through Home Assistant.
	- Added real-time control and monitoring capabilities via cloud push technology.
	- Seamless configuration flow for easy setup of the integration.

- **Documentation**
	- Updated documentation to reflect rebranding of garage door integration from "Linear" to "Nice G.O."
	- Clarified descriptions to align with the new branding and ensure consistency in device names and accounts.

### CODE REVIEW

File: source/_integrations/nice_go.markdown [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  - Cover
+  - Light
+ha_iot_class: Cloud Push
+ha_config_flow: true
+ha_codeowners:
+  - '@IceBotYT'
+ha_domain: nice_go
+ha_platforms:
+  - cover
+  - diagnostics
```
Comment (joostlek) [inline source/_integrations/nice_go.markdown:15]:
```suggestion
```
not yet

File: source/_integrations/nice_go.markdown [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+title: Nice G.O.
+description: Control Nice G.O. garage doors
+ha_release: '2024.8'
```
Comment (joostlek) [inline source/_integrations/nice_go.markdown]:
```suggestion
ha_release: '2024.9'
```

**Review by joostlek — DISMISSED** [review]

File: source/_integrations/nice_go.markdown [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  - '@IceBotYT'
+ha_domain: nice_go
+ha_platforms:
+  - cover
+  - diagnostics
+  - light
+ha_integration_type: integration
+---
+
+The Nice G.O. integration lets you control Nice G.O. garage doors through Home Assistant. Device names in Home Assistant are generated based on the names defined in your Nice G.O. mobile app.
```
Comment (c0ffeeca7) [inline source/_integrations/nice_go.markdown]:
```suggestion
The **Nice G.O.** {% term integration %} lets you control Nice G.O. garage doors through Home Assistant. Device names in Home Assistant are generated based on the names defined in your Nice G.O. mobile app.
```

File: source/_integrations/nice_go.markdown [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+ha_domain: nice_go
+ha_platforms:
+  - cover
+  - diagnostics
+  - light
+ha_integration_type: integration
+---
+
+The **Nice G.O.** {% term integration %} lets you control Nice G.O. garage doors through Home Assistant. Device names in Home Assistant are generated based on the names defined in your Nice G.O. mobile app.
+
```
Comment (c0ffeeca7) [inline source/_integrations/nice_go.markdown:21]: Are there any prerequisites? Any steps I need to do before I can get started with the integration? Install an app, have login credentials, etc?

Comment (c0ffeeca7) [inline source/_integrations/nice_go.markdown:21]: Are all Nice G.O. devices supported? Or is there a specific list of supported devices?

Comment (IceBotYT) [reply source/_integrations/nice_go.markdown:21]: All should be supported.

Comment (IceBotYT) [reply source/_integrations/nice_go.markdown:21]: Prerequisite is to have a working account with Nice G.O.

File: source/_integrations/nice_go.markdown [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+Make sure you have a working account with the Nice G.O. app and have your email and password ready.
```
Comment (c0ffeeca7) [inline source/_integrations/nice_go.markdown:24]:
```suggestion

## Prerequisites

Make sure you have a working account with the Nice G.O. app and have your email and password ready.
```

File: source/_integrations/nice_go.markdown [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+ha_platforms:
+  - cover
+  - diagnostics
+  - light
+ha_integration_type: integration
+---
+
+The **Nice G.O.** {% term integration %} lets you control Nice G.O. garage doors through Home Assistant. Device names in Home Assistant are generated based on the names defined in your Nice G.O. mobile app.
+
+
```
Comment (c0ffeeca7) [inline source/_integrations/nice_go.markdown]:
```suggestion
##
Make sure you have a working account with the Nice G.O. app and have your email and password ready.
```

File: source/_integrations/nice_go.markdown [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+ha_platforms:
+  - cover
+  - diagnostics
+  - light
+ha_integration_type: integration
+---
+
+The **Nice G.O.** {% term integration %} lets you control Nice G.O. garage doors through Home Assistant. Device names in Home Assistant are generated based on the names defined in your Nice G.O. mobile app.
+
+##
```
Comment (c0ffeeca7) [inline source/_integrations/nice_go.markdown]:
```suggestion
```

File: source/_integrations/nice_go.markdown [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+ha_platforms:
+  - cover
+  - diagnostics
+  - light
+ha_integration_type: integration
+---
+
+The **Nice G.O.** {% term integration %} lets you control Nice G.O. garage doors through Home Assistant. Device names in Home Assistant are generated based on the names defined in your Nice G.O. mobile app.
+
+Make sure you have a working account with the Nice G.O. app and have your email and password ready.
```
Comment (c0ffeeca7) [inline source/_integrations/nice_go.markdown]:
```suggestion
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @IceBotYT :+1:

**Pull request merged**